packages/paste-design-tokens/tokens/global/box-shadow.yml

```diff
@@ -5,27 +5,54 @@ imports:
   - ../aliases/box-shadow.yml
   - ../aliases/offset.yml
 props:
+  shadow-high:
+    value: "{!shadow-elevation-30}"
+    comment: High elevation default shadow.
+  shadow:
+    value: "{!shadow-elevation-20}"
+    comment: Default shadow.
+  shadow-low:
+    value: "{!shadow-elevation-10}"
+    comment: Low elevation default shadow.
   shadow-card:
     value: "{!shadow-elevation-10}"
     comment: Shadow for cards.
   shadow-focus:
-    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20}px {!palette-blue-60-transparent-80}"
-    comment: Shadow for focus ring on interactive elements
-  shadow-border-light:
+    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20}px {!palette-blue-60-transparent-70}"
+    comment: Shadow for focus ring on interactive elements.
+  shadow-focus-inverse:
+    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20}px {!palette-gray-0-transparent-40}"
+    comment: Shadow for focus ring on interactive elements on inverse backgrounds.
+  shadow-border-lighter:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-20}"
-    comment: Shadow border for disabled inputs.
+    comment: Lighter shadow border for disabled inputs.
+  shadow-border-light:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-30}"
+    comment: Light shadow border for disabled inputs.
   shadow-border:
-    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-60}"
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-50}"
     comment: Shadow border for inputs.
   shadow-border-primary-dark:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-blue-80}"
-    comment: Shadow border for inputs hover.
+    comment: Dark shadow border for inputs hover.
   shadow-border-primary-darker:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-blue-90}"
-    comment: Shadow border for inputs active.
+    comment: Darker shadow border for inputs active.
   shadow-border-error:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-red-60}"
     comment: Shadow border for error inputs.
   shadow-border-error-dark:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-red-80}"
     comment: Shadow border for error inputs hover.
+  shadow-border-inverse-darker:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-20}"
+    comment: Darker shadow border on interactive elements on inverse backgrounds.
+  shadow-border-inverse:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-50}"
+    comment: Shadow border on interactive elements on inverse backgrounds.
+  shadow-border-inverse-light:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-0-transparent-90}"
+    comment: Light shadow border on interactive elements on inverse backgrounds.
+  shadow-border-inverse-lightest:
```

Comment:
The inverse of `shadow-border-lighter` seems to be `shadow-border-inverse-lightest`, does that sound right? @serifluous 

```
shadow-border -> shadow-border-inverse
shadow-border-light -> shadow-border-inverse-light
shadow-border-lighter -> shadow-border-inverse-lightest ??
```

Is it a straight rule even? In choosing when to use what token, am I following the true sense of inverse "in a light background I want shadow-border-lighter so that translates to shadow-border-lighter"

Or is this just a product of designing a system for a subset of components? Like, will `shadow-border-darker` come at some point in the future?

Reply:
The rule is a little tricky. c/p from [this Slack discussion](https://twilio.slack.com/archives/GGQD89JRL/p1595631195316300?thread_ts=1595450472.261700&cid=GGQD89JRL
):

> on the -inverse suffix, I just had the same thought today.
> I'm justifying it like this: The token gets an -inverse suffix if it's actually an inverse of the corresponding token. The token gets an additional -light/-dark suffix if it's a shade of that inverse token (rather than an inverse of the -light/-dark default token).
> So color-text-link-inverse-darker isn't the inverse of color-text-link-darker, but is a darker shade of color-text-link-inverse...Because if it were an actual inverse of -link-darker it might then be a lighter color on the inverse palette.

As for `shadow-border-inverse-lightest`, I went with lightest as the name because the alias used is actually the lightest color (pure white) that could be on the inverse body background.

As for the missing lighter step, we can totally add that in. I didn't want to add in a token we didn't need right away, but there's no harm in doing it.

Reply:
Seems legit